tests/module.test.cpp

```diff
@@ -13,15 +16,16 @@ import beman.scope;
 //     noisy_resource()  { std::print( "construct noisy\n" ); }
 //     ~noisy_resource() { std::print( "destroy noisy\n"   ); }
 // };
+// clang-format on
```

Comment:
hmm I wonder if you caught an earlier version of my branch -- this was set to off so that clang-format didn't undo my beautiful spacing at line 22-24....